src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl

```diff
@@ -111,6 +111,34 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template match="note">
+    <xsl:variable name="type">
```

Comment:
What is the issue?  I looked at the generated html for https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html and I think I'm consistent. But I could be wrong!